woodwork/tests/accessor/test_statistics.py

```diff
@@ -1901,6 +1908,220 @@ def test_box_plot_optional_return_values(outliers_df):
+        "quantiles",
+        "low_values",
+        "high_values",
+        "low_indices",
+        "high_indices",
+        "medcouple",
+    }
+
+    expected_right_skewed_dict = {
+        "low_bound": 1.5986854923843101,
+        "high_bound": 16.945914544292435,
+        "quantiles": {
+            0.0: 1.0,
+            0.25: 3.0,
+            0.5: 4.0,
+            0.75: 6.0,
+            1.0: 30.0,
+        },
+        "low_values": [1, 1],
+        "high_values": [30],
+        "low_indices": [0, 1],
+        "high_indices": [65],
+        "medcouple": 0.3333333333333333,
+    }
+
+    expected_left_skewed_dict = {
+        "low_bound": 14.054085455707563,
+        "high_bound": 29.40131450761569,
+        "quantiles": {
+            0.0: 1.0,
+            0.25: 25.0,
+            0.5: 27.0,
+            0.75: 28.0,
+            1.0: 30.0,
+        },
+        "low_values": [1],
+        "high_values": [30, 30],
+        "low_indices": [65],
+        "high_indices": [0, 1],
+        "medcouple": -0.3333333333333333,
+    }
+
+    assert right_skewed_dict == expected_right_skewed_dict
+    assert left_skewed_dict == expected_left_skewed_dict
+
+
+def test_medcouple_outliers_with_quantiles(skewed_outliers_df):
+    outliers_series_skewed_right = skewed_outliers_df["right_skewed_outliers"]
+    outliers_series_skewed_right.ww.init()
+
+    outliers_series_skewed_left = skewed_outliers_df["left_skewed_outliers"]
+    outliers_series_skewed_left.ww.init()
+
+    override_quantiles_right = {
+        0.0: 1.0,
+        0.25: 8.0,
+        0.5: 11.0,
+        0.75: 25.0,
+        1.0: 30.0,
+    }
+
+    override_quantiles_left = {
+        0.0: 1.0,
+        0.25: 3.0,
+        0.5: 4.0,
+        0.75: 26.0,
+        1.0: 30.0,
+    }
+
+    expected_skewed_dict = {
+        "low_bound": 1.0,
+        "high_bound": 30.0,
+        "quantiles": None,
+        "low_values": [],
+        "high_values": [],
+        "low_indices": [],
+        "high_indices": [],
+        "medcouple": 0.3333333333333333,
+    }
+
+    right_skewed_dict = outliers_series_skewed_right.ww.medcouple_dict(
+        quantiles=override_quantiles_right,
+    )
+    left_skewed_dict = outliers_series_skewed_left.ww.medcouple_dict(
+        quantiles=override_quantiles_left,
+    )
+
+    expected_skewed_dict["quantiles"] = override_quantiles_right
+    assert right_skewed_dict == expected_skewed_dict
+    expected_skewed_dict["quantiles"] = override_quantiles_left
+    expected_skewed_dict["medcouple"] = -0.3333333333333333
+    assert left_skewed_dict == expected_skewed_dict
+
+
+def test_get_outliers_for_column_with_nans_medcouple(skewed_outliers_df):
+    contains_nans_series = skewed_outliers_df["has_outliers_with_nans"]
+    contains_nans_series.ww.init()
+
+    medcouple_dict = contains_nans_series.ww.medcouple_dict()
+
+    expected_skewed_dict = {
+        "low_bound": 1.9490141192882326,
+        "high_bound": 13.459435908219328,
+        "quantiles": {
```

Comment:
do we do any testing with custom quantiles?

Reply:
We do! The logic for it comes before the bounds or medcouple calculation so its method independent, but I went ahead and changes a test to account for the medcouple approach.